spec/source_finder/source_file_globber_spec.rb

```diff
@@ -103,9 +104,9 @@
   describe '#ruby_files_arr' do
     before do
       expect(globber).to receive(:glob)
-        .with('{Rakefile,{*,.*}.{rb,rake,gemspec},' \
-              '{src,app,config,db,lib,test,spec,feature}/**/{*,.*}.' \
-              '{rb,rake,gemspec}}')
+        .with('{Rakefile,{*,.*}.{gemspec,rake,rb},' \
+              '{app,config,db,feature,lib,spec,src,test}/**/{*,.*}.' \
+              '{gemspec,rake,rb}}')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.